sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_cbs_async.py

```diff
@@ -87,7 +88,6 @@ async def _put_token(
             operation=CBS_PUT_TOKEN,
             type=token_type,
         )
-        self._mgmt_link.next_message_id += 1
```

Comment:
so this was resetting with every mgmt operation?

Reply:
yes